src/main/java/org/jitsi/srtp/crypto/Aes.java

```diff
@@ -188,13 +189,9 @@ private static BlockCipherFactory benchmark(
 
                     long startTime = System.nanoTime();
 
-                    for (int inOff = 0;
-                            inOff < inEnd;
```

Comment:
I think this leaves `inEnd` unused

Reply:
Yes, I removed it in a subsequent commit.